app/assets/styles/tools/main.css

```
@@ -0,0 +1,8 @@
+@keyframes fadeIn {
```

Stylelint (17.4.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)